### PR TITLE
Refactor item related enums and move them to separate file.

### PR DESCRIPTION
**Pull request by Predelnik**
This should be mostly no-op PR to ease future working with items. Some unused effects calculation was removed for simplicity.
However I also attached some minor changes/bugfixes:
1. Hotkeys for opening panels.
2. Fix double creation of dialogs.
3. Support ability to heal for Pepin.

### CODE REVIEW

File: apps/celview/CMakeLists.txt [hunk 1]
```diff
@@ -2,5 +2,10 @@ add_executable(celview
     main.cpp
 )
 
-target_link_libraries(celview NuklearMisc SDL2::SDL2main)
+target_link_libraries(celview NuklearMisc)
+if (MINGW)
+target_link_libraries(celview -lmingw32 SDL2::SDL2main)
```
Comment (wheybags) [inline apps/celview/CMakeLists.txt]: Is this only needed for celview?

File: apps/freeablo/faworld/actor.cpp [hunk 2]
```diff
@@ -146,6 +146,8 @@ namespace FAWorld
         }
     }
 
+    void Actor::heal() { mStats.mHp = 100; }
```
Comment (wheybags) [inline apps/freeablo/faworld/actor.cpp]: I'd do mStats.mHp = mStats.mHp.max;

File: apps/celview/CMakeLists.txt (hunk 1 above)
Comment (Predelnik) [reply apps/celview/CMakeLists.txt]: Also for launcher. Everything else doesn't include SDL2main

File: apps/freeablo/faworld/actor.cpp (hunk 2 above)
Comment (Predelnik) [reply apps/freeablo/faworld/actor.cpp]: Ok

File: apps/celview/CMakeLists.txt (hunk 1 above)
Comment (Predelnik) [reply apps/celview/CMakeLists.txt]: Hopefully it will be resolved upstream eventually then fix can be removed.

**Pull request merged**